internal/usecases/reports/get_assets_report.go

```diff
@@ -0,0 +1,153 @@
+		pegoutProvider:   pegoutProvider,
+		peginRepository:  peginRepository,
+		pegoutRepository: pegoutRepository,
+	}
+}
+
+func (useCase *GetAssetsReportUseCase) Run(ctx context.Context) (pkg.GetAssetsReportResponse, error) {
+	response := pkg.GetAssetsReportResponse{
+		BtcBalance:    entities.NewWei(0).AsBigInt(),
+		RbtcBalance:   entities.NewWei(0).AsBigInt(),
+		BtcLocked:     entities.NewWei(0).AsBigInt(),
+		RbtcLocked:    entities.NewWei(0).AsBigInt(),
+		BtcLiquidity:  entities.NewWei(0).AsBigInt(),
+		RbtcLiquidity: entities.NewWei(0).AsBigInt(),
+	}
+	btcBalance, err := useCase.GetBtcBalance()
+	if err != nil {
+		return response, err
+	}
+	response.BtcBalance = btcBalance.AsBigInt()
+	rbtcBalance, err := useCase.GetRBTCBalance(ctx)
+	if err != nil {
+		return response, err
+	}
+	response.RbtcBalance = rbtcBalance.AsBigInt()
+
+	rbtcLocked, err := useCase.GetRBTCLocked(ctx)
+	if err != nil {
+		return response, err
+	}
+	response.RbtcLocked = rbtcLocked.AsBigInt()
+
+	lockedBtc, err := useCase.GetBTCLocked(ctx)
+	if err != nil {
+		return response, err
+	}
+	response.BtcLocked = lockedBtc.AsBigInt()
+
+	btcLiquidity, err := useCase.GetBTCLiquidity(ctx)
+	if err != nil {
+		return response, err
+	}
+	response.BtcLiquidity = btcLiquidity.AsBigInt()
+
+	rbtcLiquidity, err := useCase.GetRBTCLiquidity(ctx)
+	if err != nil {
+		return response, err
+	}
+	response.RbtcLiquidity = rbtcLiquidity.AsBigInt()
+	return response, nil
+}
+
+func (useCase *GetAssetsReportUseCase) GetRBTCLiquidity(ctx context.Context) (*entities.Wei, error) {
+	rbtcLiquidity, err := useCase.peginProvider.AvailablePeginLiquidity(ctx)
+	if err != nil {
+		return nil, err
+	}
+	return rbtcLiquidity, nil
+}
+
+func (useCase *GetAssetsReportUseCase) GetBTCLiquidity(ctx context.Context) (*entities.Wei, error) {
+	btcLiquidity, err := useCase.pegoutProvider.AvailablePegoutLiquidity(ctx)
+	if err != nil {
+		return nil, err
+	}
+	return btcLiquidity, nil
+}
+
+func (useCase *GetAssetsReportUseCase) GetBTCLocked(ctx context.Context) (*entities.Wei, error) {
+	lockedPegout := entities.NewWei(0)
+	quotes, err := useCase.pegoutRepository.GetRetainedQuoteByState(ctx,
+		quote.PegoutStateWaitingForDeposit, quote.PegoutStateWaitingForDepositConfirmations,
+	)
+	if err != nil {
+		return nil, err
+	}
+	for _, retainedQuote := range quotes {
+		lockedPegout.Add(lockedPegout, retainedQuote.RequiredLiquidity)
+	}
+
+	return lockedPegout, nil
+}
+
+func (useCase *GetAssetsReportUseCase) GetRBTCLocked(ctx context.Context) (*entities.Wei, error) {
+	lockedPegin := entities.NewWei(0)
+	peginQuotes, err := useCase.peginRepository.GetRetainedQuoteByState(ctx, quote.PeginStateWaitingForDeposit)
+	if err != nil {
+		return nil, err
+	}
+	for _, retainedQuote := range peginQuotes {
+		lockedPegin.Add(lockedPegin, retainedQuote.RequiredLiquidity)
+	}
+	pegoutQuotes, err := useCase.pegoutRepository.GetRetainedQuoteByState(ctx, quote.PegoutStateRefundPegOutSucceeded)
```

Comment:
this doesn't count as locked, it is true that is not counted as liquidity neither, but we're not saving those RBTC for any user, they are just waiting for rebalance